gems/actionpack/CVE-2021-22942.yml

```diff
@@ -0,0 +1,70 @@
+---
+gem: actionpack
+framework: rails
+cve: 2021-22942
+url: https://groups.google.com/g/rubyonrails-security/c/wB5tRn7h36c
+title: Possible Open Redirect in Host Authorization Middleware
+date: 2021-08-19
+description: |
+  There is a possible open redirect vulnerability in the Host Authorization middleware in Action Pack. This vulnerability has been assigned the CVE identifier CVE-2021-22942.
```

Comment:
I'm doing this from mobile, so hard for me to count, but several of these lines seem pretty long. We prefer wrapping somewhere in the 80-100 character range (see other files). Can you check and see if any of these are too long and need to be wrapped?

Reply:
Serves me right for doing a ninja edit in the github editor :)